### PR TITLE
NOTIF-109 Daily emails

### PR DESCRIPTION
**Pull request by josejulio**

### CODE REVIEW

**Comment by josejulio** [comment]
@pilhuhn This one is ready for review

File: src/test/resources/application.properties [hunk 1]
```diff
@@ -33,3 +33,5 @@ processor.email.bop_apitoken=addme
 processor.email.bop_client_id=policies
 processor.email.bop_env=qa
 processor.email.no_reply=no-reply@redhat.com
+
+quarkus.scheduler.enabled=false
```
Comment (pilhuhn) [inline src/test/resources/application.properties:37]: Shouldn't that be enabled?

Comment (josejulio) [reply src/test/resources/application.properties:37]: This is a test setting, I think is fine disabled.

**Review by pilhuhn — APPROVED** [review]
Lgtm

**Pull request merged**